docs/migration.txt

```diff
@@ -153,3 +153,10 @@ on the FilterSet:
 
 To fix this, simply remove the methods from your class. You can subclass
 ``OrderingFilter`` to migrate any custom logic.
+
+
+Deprecated ``FILTERS_HELP_TEXT_FILTER`` and ``FILTERS_HELP_TEXT_EXCLUDE`` settings
+----------------------------------------------------------------------------------
+Details: https://github.com/carltongibson/django-filter/pull/484
```

Comment:
Hi. I've updated the PR with more details. This PR is prep for #437, which includes more descriptive filter labels. eg, "Published date is less than" and "Exclude author name starts with". The `help_text` would no longer be necessary.